src/game/engine/house.h

```diff
@@ -174,9 +176,13 @@ class HouseClass
     int Currently_Owned_Aircraft_Count() const { return m_CurrentAircraftCount; }
     coord_t Base_Center() const { return m_BaseCenter; }
     RegionClass *Threat_Regions() { return m_ThreatRegions; }
+    target_t Chrono_Object() const { return m_ChronoObject; }
     void Code_Pointers() {}
     void Decode_Pointers() {}
-
+    uint32_t BScan_Build_Count() const { return m_BScan.m_HaveBuilt; }
```

Comment:
This should be something like
`inline bool Has_Buildings() { return m_BScan.m_HaveBuilt != 0; }`
because the trackers are all bitfields, likewise functions that can check the fields is needed something like
```
inline bool Has_Built(TYPE type) { return SCAN.m_HaveBuilt & type != 0; }
inline bool Does_Human_Own(TYPE type) { return SCAN.m_HumanPrereqs & type != 0; }
inline bool Does_AI_Own(TYPE type) { return SCAN.m_AIPrereqs & type != 0; }
```